features/cadastro_e_manutenção_de_clientes.feature

```diff
@@ -0,0 +1,236 @@
+Feature: Cadastro e manutenção de clientes (inserir, atualizar e remover)
+Como um usuário do app de delivery
```

Comment:
As palavras-chave tem que estar em inglês pra linguagem de teste funcionar, então tem que trocar pra "As a", "Given", "And", "Then", etc.

Reply:
Os termos já foram substituídos.